src/styles/buttons.less

```diff
@@ -127,6 +127,7 @@
 
         & + * {
             margin-left: 4px;
+            padding-left: 5px;
```

Comment:
Wait, I got confused. This is the library stylesheet, no changes should be made here at all. Let's remove this.

Reply:
removed